packages/twenty-emails/src/emails/clean-suspended-workspace.email.tsx

```diff
@@ -0,0 +1,42 @@
+import { BaseEmail } from 'src/components/BaseEmail';
+import { CallToAction } from 'src/components/CallToAction';
+import { MainText } from 'src/components/MainText';
+import { Title } from 'src/components/Title';
+
+type CleanSuspendedWorkspaceEmailProps = {
+  inactiveDaysBeforeDelete: number;
+  userName: string;
+  workspaceDisplayName: string | undefined;
+};
+
+export const CleanSuspendedWorkspaceEmail = ({
+  inactiveDaysBeforeDelete,
+  userName,
+  workspaceDisplayName,
+}: CleanSuspendedWorkspaceEmailProps) => {
+  const helloString = userName?.length > 1 ? `Hello ${userName}` : 'Hello';
+
+  return (
+    <BaseEmail width={333}>
+      <Title value="Deleted Workspace 🥺" />
+      <MainText>
+        {helloString},
+        <br />
+        <br />
+        We wanted to inform you that your <b>{workspaceDisplayName}</b>{' '}
```

Comment:
Your workspace <b>{workspaceDisplayName}</b> has been deleted as it remained inactive for {inactiveDaysBeforeDelete} days.

All data in this workspace has been permanently deleted.

If you wish to use Twenty again, you can create a new workspace.